index.html

```diff
@@ -1,182 +1,181 @@
+            <li class="lang2">CSS</li>
+            <li class="lang3">JavaScript</li>
+        </ul>
+        <div class="see">
+        <a href="#" class="text">See Project</a>
+        </div>
+    </div>
+    </section>
+
+    <!-- About me Section -->
+    <section class="about">
+        <div class="base">
+            <div class="top">
+        <h2>About Myself</h2>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="social">
+            <h3>LET'S CONNECT</h3>
+            <ul class="icons">
+                <li><a href="#"><img src="./image/Vector.svg"></a></li>
+                <li><a href="#"><img src="./image/Linkedin_icon.svg"></a></li>
+                <li><a href="#"><img src="./image/Vector1.svg"></a></li>
+                <li><a href="#"><img src="./image/Vector2.svg"></a></li>
+                <li><a href="#"><img src="./image/Vector3.svg"></a></li>
+            </ul>
+        </div>
+        <div class="btn">
+            <a href="#" class="resume">Get my resume</a>
+        </div>
+        <div class="skill_list">
+            <div class="title1">
+                <h2>Languages</h2>
+                <a href="#" class= "fa fa-angle-down"></a>
+                <ul class="langlist">
+                <li class="list1">
+                    <img src="./image/HTML.png" alt="HTML icon">
+                    <p>JavaScript</p>
+                </li>
+                <li class="list2">
+                    <img src="./image/CSS.png" alt=" CSS icon">
+                    <p>HTML</p>
+                </li>
+                <li class="list3">
+                    <img src="./image/JS.png" alt="JS icon">
+                    <p>CSS</p>
+                </li>
+                </ul>
+            </div>
+            <div class="title2">
+                <h2>Frameworks</h2>
+                <a href="#" class= "fa fa-angle-right"></a>
+            </div>
+            <div class="title3">
+                <h2>Skills</h2>
+                <a href="#" class= "fa fa-angle-right"></a>
+            </div>
+        </div>
+        </div>
+    </section>
+
+    <!-- Contact me -->
+    <section class="contact">
+        <h4>Contact me</h4>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I’d love to help with it! </p>
+    <form action="https://formspree.io/f/{form_id}" method="post">
+        <input type="text" id="fname" name="fname" value="First Name" maxlength="30" required>
+        <input type="email" id="email" name="email" value="Your Email" required>
+        <input type="text" id="msg" name="msg" value="Write your message here..." maxlength="500" required>
+        <button href="#" class="submit">Get in touch</a>
```

Comment:
- [ ] I would kindly suggest you to Please check the `network` tab in your browser dev tools to confirm that the form is submitted successfully with valid input values.
- [ ]  form not successfully submit,Please, open the form in the browser and verify if all validations described in the project description are working on form submission,as this will proof your professionalism please fix it.
![image](https://github.com/Nomarg/My-Portfolio/assets/92458236/5a751860-3345-4a85-bc3c-f07a367d75c3)